src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -414,6 +414,7 @@ internal enum WellKnownMember
         System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames,
 
         System_String__Format_IFormatProvider,
+        System_String__Substring,
```

Comment:
>System_String__Substring, [](start = 8, length = 25)

Members of special types are special members, not well known.

Reply:
What about the Format overload above?

Reply:
> What about the Format overload above?

Should be special too.

---
In reply to: [274721553](https://github.com/dotnet/roslyn/pull/34897#discussion_r274721553) [](ancestors = 274721553)

Reply:
Doesn't matter to me, I'll move this one at least :)